packages/terra-badge/tests/jest/__snapshots__/Badge.test.jsx.snap

```diff
@@ -1,20 +1,20 @@
 exports[`test should render a badge component in the order, icon and text with medium size 1`] = `
 <span
-  className="terra-Badge terra-Badge--medium terra-Badge--default">
+  className="badge medium default">
   <img
     alt="Test icon" />
   <span
-    className="terra-Badge-text">
+    className="text">
```

Comment:
> The jest tests have access to the CSS classes before they run through css modules

This is partially true, Jest actually mocks out the style module so that the class name is the property be accessed off the style object. For CSS modules, this coincidentally is the original class name.